sqlboiler.toml

```diff
@@ -7,6 +7,6 @@ blacklist=["schema_migrations"]
   dbname="mydb"
   host="localhost"
   port=5432
-  user="user"
-  pass="password"
+  user="postgres"
```

Comment:
why change that ?
this is also used in docker-compose.yml

Reply:
I was not unestand why its on the git, 
add to it .gitignore
push changes by mistake